package.json

```diff
@@ -8,15 +8,22 @@
     "build": "npm run clean && webpack",
     "start": "styleguidist server",
     "styleguide": "styleguidist build",
-    "test": "npm run eslint:src && jest --coverage",
+    "test": "npm run eslint:src && jest --coverage && npm run size",
     "eslint:src": "eslint ./src ./examples ./*.js",
     "commitmsg": "commitlint -e",
     "precommit": "lint-staged",
     "prepublish": "NODE_ENV=production npm run build && jest",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "logo": "cd logo && sketchtool export artboards logo.sketch",
-    "imagemin": "imagemin --out-dir=logo --plugin=pngquant --plugin=svgo"
+    "imagemin": "imagemin --out-dir=logo --plugin=pngquant --plugin=svgo",
+    "size": "size-limit"
   },
+  "size-limit": [
+    {
+      "path": "dist/index.js",
+      "limit": "5.76 KB"
```

Comment:
I'm wondering if there is any way this can be reduced?

Reply:
Want me to work on trying to get this smaller a little?

Reply:
We could try but let's do in a separate PR.

Reply:
I'm gonna do the why and merge this then, cool?

Reply:
Yes, please!

Reply:
@okonet Done ! Just gonna wait for the ci